Software/Development/SemesterProject/App/controlTask.c

```diff
@@ -17,15 +17,21 @@ void executeControl(void)
     if (readSystemState() == RUNNING)
     {
         if (isOpenLoopControlSelected())
+        {
             // Call open-loop function
             openLoopVFControl();
+        }
 
-        // else
+         else
+         {
+             enableDrivers();

```

Comment:
Im not sure whethere it's the best idea to leave this here, maybe add a comment to make sure that this is to be taken out and it's only for testing purposes ¿?

Reply:
I remove it